pytest_fixtures/component/oscap.py

```diff
@@ -20,26 +20,32 @@ def tailoring_file_path(session_target_sat):
 
 
 @pytest.fixture(scope="session")
-def oscap_content_path(session_target_sat):
+def oscap_content_source(session_target_sat):
+    """Get scap content file path"""
+    return f'{settings.oscap.content_dir}/ssg-rhel{session_target_sat.os_distribution_version.split(".")[0]}-ds.xml'
```

Comment:
```suggestion
    return f'{settings.oscap.content_dir}/ssg-rhel{session_target_sat.os_version.major}-ds.xml'
```
Same suggestion for all the other `os_distribution_version` occurrences.